src/NServiceBus.AwsLambda.SQS.Analyzer/NServiceBus.AwsLambda.SQS.Analyzer.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(AnalyzerTargetFramework)</TargetFramework>
+    <IsRoslynComponent>true</IsRoslynComponent>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\NServiceBus.snk</AssemblyOriginatorKeyFile>
+    <IsPackable>false</IsPackable>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Particular.Packaging" Version="2.2.0" PrivateAssets="All" />
+    <!-- this package can only be upgraded when we drop support for older versions of Visual Studio -->
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.11.0" />
+  </ItemGroup>
+  
```

Comment:
Core analyzer includes [this](https://github.com/Particular/NServiceBus/blob/release-8.1/src/NServiceBus.Core.Analyzer/NServiceBus.Core.Analyzer.csproj#L17-L21C12). Do we need it here?

Reply:
I checked in code and all analyzers have this, as well as the generator projects. I've added it here as well as to the AzureFunctions projects